docs/content/latest/yugabyte-cloud/cloud-monitor/cloud-alerts.md

```diff
@@ -0,0 +1,141 @@
+
+High CPU use could also indicate a problem and may require debugging by Yugabyte Support.
+
+If CPU use is continuously higher than 80%, your workload may also have exceeded the capacity of your cluster. Consider scaling your cluster by adding vCPUs. Refer to [Scale and configure clusters](../../cloud-clusters/configure-clusters/).
+
+#### Fix database overload alerts
+
+A notification is sent for the following database overload alert:
+
+- Cluster queues overflow and/or compaction overload.
+<!-- Cluster exceeds 200 simultaneous YSQL connections.-->
+
+This alert is triggered if any one of the following conditions occurs:
+
+- The number of client requests that timed out while waiting in the service queue is greater than 0 for 10 minutes. This can be caused by your application generating requests (such as queries and scans) at a rate faster than the database can process them.
+- The number of client requests that were dropped because the service queue was full is greater than 0 for 10 minutes. This can be caused by your application generating requests (such as queries and scans) at a rate faster than the database can process them.
+- The number of compaction requests that were rejected is greater than 0 for 10 minutes. This can be caused by running a high INSERT/ UPDATE/ DELETE workload on a cluster that is undersized.
+
+Cluster queue overflow and compaction overload are indicators of the incoming traffic and system load. If the backends get overloaded, requests pile up in the queues. When the queue is full, the system responds with backpressure errors. This can cause performance degradation.
+
+If your cluster is generating this alert, you may need to rate limit your queries. Look at optimizing the application generating the requests in the following ways:
+
+- Examine the application retry logic, and reduce total retry count.
+- Increase client request timeout at the driver layer.
+- Reduce query parallelism.
+- If needed, set the driver execution to "sync" rather than "async" to serialize queries.
+
+If your cluster is generating this alert and isn’t under a very large workload, contact [Yugabyte Support](https://support.yugabyte.com/hc/en-us/requests/new?ticket_form_id=360003113431).
+
+### Billing alerts
+
+Billing alerts can trigger for the following events:
+
+| Alert | Response |
+| :--- | :--- |
+| Credit card expiring within 3 months | Add a new credit card. |
+| Invoice payment failure | Check with your card issuer to ensure that your card is still valid and has not exceeded its limit. Alternatively, you can add another credit card to your billing profile. |
+| Credit point usage exceeds the 50%, 75%, and 90% limit | Add a credit card to your billing profile, or contact Yugabyte Support. |
```

Comment:
Because we can do this now! 👍 

```suggestion
| Credit point usage exceeds the 50%, 75%, and 90% limit | Add a credit card to your billing profile, or contact {{<support-cloud>}}. |
```